### PR TITLE
improved getting started docs

### PR DESCRIPTION
**Pull request by tmbo**
**Proposed changes**:
- remove leading `$ ` from bash code blocks (as it gets copied when you click the copy button which is very annoying). if we feel we need it, it should be implemented in the highlighter for bash as a displayed but not copied char
- made the docs landing page a little more useful with a pointer where to go next
- cleaned up the installation page
   - removed TOC (not necessary, we got a sidebar now)
   - added a hint to run `rasa init` after installation or use prototyping page
   - use groups for the tab sections (selecting "MacOS" once will show you macos for all tab groups)
   - always use `pip3` and `python3` as it is a best practice
   - made extra dependencies headlines more useful in sidebar

**Status (please check what you already did)**:
- [x] updated the documentation

### CODE REVIEW

File: docs/docs/installation.mdx [hunk 1]
```diff
@@ -245,7 +246,7 @@ anywhere. If you want to use MITIE, you need to
 tell it where to find this file (in this example it was saved in the
 `data` folder of the project directory).
 
-:::caution
+:::caution Deprecation Warning
 Mitie support is likely to be deprecated in a future release.
```
Comment (erohmensing) [inline docs/docs/installation.mdx:250]: Is this actually true? we keep saying so and then deciding to keep it around

File: docs/docs/messaging-and-voice-channels.mdx [hunk 2]
```diff
@@ -12,7 +12,7 @@ in a `credentials.yml` file.
 An example file is created when you run `rasa init`, so it's easiest to edit that file
 and add your credentials there. Here is an example with Facebook credentials:
 
-```yaml
+```yaml title="credentials.yml"
```
Comment (erohmensing) [inline docs/docs/messaging-and-voice-channels.mdx:15]: Very cool, didn't know we could do this 😎

File: docs/docs/installation.mdx (hunk 1 above)
Comment (tmbo) [reply docs/docs/installation.mdx:250]: I mean you got a very valid point there, but as far as I remember there is no alternative for certain languages (was it Japanese or Cantonese 🤷 ). let's poll the channel.

File: docs/docs/messaging-and-voice-channels.mdx (hunk 2 above)
Comment (tmbo) [reply docs/docs/messaging-and-voice-channels.mdx:15]: yes it's a lot nicer than our usual: "... into your `xyz.yaml` put: "

File: docs/docs/installation.mdx (hunk 1 above)
Comment (tmbo) [reply docs/docs/installation.mdx:250]: I'll remove it for now, until we have a plan to deprecate it.

**Pull request merged**